### PR TITLE
Import API v2 docs

### PR DESCRIPTION
**Pull request by shiroyasha**

### CODE REVIEW

**Comment by shiroyasha** [comment]
@ijovan Generated the middleman headers. Now it should work 👍

**Comment by ijovan** [comment]
`loyout` => `layout` 😀

**Comment by shiroyasha** [comment]
:D fixed it 👌

**Comment by ijovan** [comment]
Looks good now.

The links leading from the contents page to individual entities is not right, though. It's missing the `/api_v2/`.

File: source/post.html.erb [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <div class="f4 pv3-ns pa4-l">
 
       <a href="#docs-nav" class="f6 dn-l dib link-reset black-50 bg-black-05 hover-bg-black-10 bg-animate pv1 ph3 mb3 br2">Docs Navigation</a>
-      <h2 class="f3 f2-m lh-title mv2"><a href="#" class="link-reset underline-hover article-title"><%= current_page.data.title %></a></h2>
+      <h2 class="f3 f2-m lh-title mv2"><a href="#" class="link-reset underline-hover article-title"><%= current_page.data.title || yield_content(:api_v2_title) %></a></h2>
```
Comment (shiroyasha) [inline source/post.html.erb]: Damn, I forgot to fix this.

**Pull request merged**